Project.toml

```diff
@@ -10,7 +10,7 @@ Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 
 [compat]
 Intervals = "1.3"
-julia = "1"
+julia = "1.3"
```

Comment:
This package has never actually actively supported 1.0.
Like all the stuff close source.
Arguably we should retroactively apply those bounds when moving the registry